src/TripSegment/index.js

```diff
@@ -0,0 +1,282 @@
+import CarrierLogo, { StyledCarrierLogo } from "../CarrierLogo";
+import Airplane from "../icons/Airplane";
+import Bus from "../icons/Bus";
+import Train from "../icons/Train";
+import Text from "../Text";
+import { Item } from "../List/ListItem";
+
+import type { Props, State } from "./index";
+
+// TODO: Fix animations
+const ToggleUp = keyframes`
+  from { max-height: 0%; overflow: hidden; opacity: 0 }
+  to { max-height: 100%; overflow: visible; top: 0; opacity: 1 }
+`;
+
+const ToggleDown = keyframes`
+  0% { max-height: 0;  overflow: hidden; opacity: 1}
+  75% { opacity: 0.4; max-height: 50% }
+  100%  { max-height: 100%; overflow: visible; opacity: 0}
+`;
+
+const Milestone = styled.div`
+  display: flex;
+  height: 55px;
+  align-items: center;
+  padding-right: ${({ theme }) => theme.orbit.spaceXSmall};
+`;
+
+Milestone.defaultProps = {
+  theme: defaultTokens,
+};
+
+export const StyledSegment = styled(({ className, children }) => (
+  <div className={className}>{children}</div>
+))`
+  display: flex;
+  font-family: ${({ theme }) => theme.orbit.fontFamily};
+  font-size: ${({ theme }) => theme.orbit.fontSizeTextNormal};
+  flex-direction: row;
+  flex-wrap: wrap;
+  width: 100%;
+  padding: ${({ theme }) => theme.orbit.spaceXSmall};
+  background: ${({ theme }) => theme.orbit.backgroundCard};
+  border-radius: ${({ theme }) => theme.orbit.borderRadiusNormal};
+  border-width: ${({ theme }) => theme.orbit.borderWidthCard};
+  border-style: ${({ theme }) => theme.orbit.borderStyleCard};
+  border-color: ${({ theme }) => theme.orbit.paletteCloudNormal};
+  position: relative;
+  &:hover {
+    cursor: pointer;
+  }
+  &:before {
+    position: absolute;
+    right: 100%;
+    top: 27px;
+    content: " ";
+    width: 0;
+    height: 0;
+    border-style: solid;
+    border-width: 6px 6px 6px 0;
+    border-color: transparent ${({ theme }) => theme.orbit.paletteCloudNormal} transparent
+      transparent;
+    margin-top: -7px;
+  }
+  &:after {
+    border-color: transparent ${({ theme }) => theme.orbit.paletteWhite} transparent transparent;
+    width: 0;
+    height: 0;
+    border-style: solid;
+    border-width: 4px 4px 4px 0;
+    border-color: transparent ${({ theme }) => theme.orbit.paletteWhite} transparent transparent;
+    margin-top: -5px;
+    content: " ";
+    position: absolute;
+    right: 100%;
+    top: 27px;
+  }
+`;
+
+StyledSegment.defaultProps = {
+  theme: defaultTokens,
+};
+
+const Flights = styled.div`
+  display: flex;
+  flex-direction: column;
+`;
+
+const Wrapper = styled.div`
+  display: flex;
+`;
+
+const WrapperCarrier = styled.div`
+  display: flex;
+  align-items: center;
+`;
+
+const WrapperInner = styled.div`
+  display: flex;
+  width: 100%;
+  justify-content: space-between;
+  align-items: center;
+  ${StyledCarrierLogo} {
+    height: 24px;
+    width: 24px;
+    padding-left: ${({ theme }) => theme.orbit.spaceXSmall};
+    padding-right: ${({ theme }) => theme.orbit.spaceXXSmall};
```

Comment:
Move this padding to `StyledSegment` => padding: 8px 12px 8px 8px

Reply:
there is another styledsegment in list, suppose it will be better to left it as it is.